tests/unit/explore_test.sh

```diff
@@ -67,9 +67,18 @@ function test_explore_files_under_git_repo()
   assertEquals "($LINENO)" "$MSG_OUT" "$output"
 
   output=$(explore_main 'GNU grep' '.' 'TEST_MODE')
-  expected_result="git grep -e 'GNU grep' -nI ."
+  expected_result="git grep --context 0 -e 'GNU grep' --line-number -I ."
   assertEquals "($LINENO)" "$expected_result" "$output"
 
+  # Test for non zero context value
+  output=$(explore_main --show-context=5 'GNU grep' '.' 'TEST_MODE')
+  expected_result="git grep --context 5 -e 'GNU grep' --line-number -I ."
+  assertEquals "($LINENO)" "$expected_result" "$output"
+
+  MSG_OUT='7'
+  output=$(explore_main --show-context=3 'avoid' codestyle_error.c | wc -l)
+  assertEquals "($LINENO)" "$MSG_OUT" "$output"
```

Comment:
Counting the number of lines is good, but you could also add some tests that check for the string before and after the match. You can leverage some of the functions in the `test/unit/utils.sh`; for example, `assert_substring_match` could be handy for this type of test, but we have other options.